### ISSUE TITLE
Dynamic step navigation and file uploads

### ISSUE THREAD

**Issue by trottera** [craue/CraueFormFlowBundle#247]
Hi,

i've created a form flow with allowDynamicStepNavigation and handleFileUploads set to true. One step of the form contains a file upload. 

If i'm stepping through the form flow there's no problem. But when i jump back to the form step, where the file gets uploaded, and i don't add a file to the file field the previous uploaded file gets replaced by a null value. 

I think, if no new file gets uploaded through the form, the old file should remain in the form. Is there a way to accomplish this?

**Comment by craue** [comment]
Then there would be no way to get rid of the uploaded file.

**Comment by trottera** [comment]
imo if the intention would be to delete the previously uploaded file it would be more user-friendly to provide a deletion link. If you want to replace the existing file, you upload a new file. And if you want to keep the existing file, you leave the file field blank.

Is this possible?

**Comment by craue** [comment]
For me, going back within the flow means you want to change something. So if there's no upload provided, then there should be none. But you could try https://github.com/craue/CraueFormFlowBundle/issues/83#issuecomment-111213429.

**Comment by trottera** [comment]
That's true. Although i think that the file input field is a special case, since there's no way to populate it with the previous submitted data. For every other input type the user has to take action to delete his previous submitted data, like clearing a textfield, or setting the value of a select field. Only for the file field he has to take action, to keep the previously submitted data. 

Anyways, thanks for the help. I'll try this.

**Comment by xuko** [comment]
I'm trying to do the same, keep the files, when I go to previous step.
@trottera did you get a solution?
